apps/etterlatte-vedtaksvurdering/src/main/kotlin/vedtaksvurdering/Vedtak.kt

```diff
@@ -47,6 +47,28 @@ sealed class VedtakFelles(
     open val attestasjon: Attestasjon? = null,
 )
 
+data class VedtakSammendrag(
```

Comment:
Brukes denne kun for å få et sammendrag av de to andre vedtakstypene?

Reply:
Ja. jeg ville egentlig bruke VedtakFelles siden den er akkurat det jeg trengte. Men kan jo ikke det når den er seal.
Ble litt usikker hva som var best å gjøre. Åpen for forslag.

Reply:
Vi burde jo rydde hele vedtak egentlig